modules/ROOT/pages/bolt/structure-semantics.adoc

```diff
@@ -458,10 +464,35 @@ Point3D::Structure(
 
 Where the `srid` is a _Spatial Reference System Identifier_.
 
+[[structure-vector]]
+=== `Vector`
+label:new[Introduced in bolt 6.0]
+
+An ordered collection of homogeneous types, representing a vector in a multidimensional space.
```

Comment:
```suggestion
An ordered collection of homogeneous types.
```
A vector lives in a multidimensional space by default, and if we take away that part, it remains 

> An ordered collection of homogeneous types, representing a vector.

... and saying that a vector represents a vector is just a little redundant :D